framework/docker/chain_node.go

```diff
@@ -353,40 +353,16 @@ func (tn *ChainNode) createNodeContainer(ctx context.Context) error {
 	chainCfg := tn.cfg.ChainConfig
 
 	cmd := []string{chainCfg.Bin, "start", "--home", tn.homeDir}
-	if len(chainCfg.AdditionalStartArgs) > 0 {
-		cmd = append(cmd, chainCfg.AdditionalStartArgs...)
+	if startArgs := tn.getAdditionalStartArgs(); len(startArgs) > 0 {
+		cmd = append(cmd, startArgs...)
 	}
 
 	usingPorts := nat.PortMap{}
 	for k, v := range sentryPorts {
 		usingPorts[k] = v
 	}
-	for _, port := range chainCfg.ExposeAdditionalPorts {
-		usingPorts[nat.Port(port)] = []nat.PortBinding{}
-	}
```

Comment:
removing this as it was just left in from the original interchaintest code, if it's useful we can add it back.